bicep/aks-cluster.bicep

```
@@ -0,0 +1,75 @@
+param dnsPrefix string
+param linuxAdminUsername string
+@secure()
+param sshRSAPublicKey string
+param subnetId string
+param location string = resourceGroup().location
+
+resource aks 'Microsoft.ContainerService/managedClusters@2023-01-01' = {
+  name: 'pocimpinfak1401'
```

Per naming convention policy, let's make these uppercase for consistency

```suggestion
  name: 'POCIMPINFAK1401'
```